lib/dynflow/execution_plan/steps/error.rb

```diff
@@ -1,5 +1,7 @@
 module Dynflow
   module ExecutionPlan::Steps
+    class PlanCancelledError < Dynflow::Error; end
```

Comment:
What's the plan with this error class?

Reply:
None, it's a vestige from the first implementation of "cancelled" state.